schema_based_on_diagram.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name varchar(100),
+    date_of_birth date
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patients_id INT,
+    status varchar(255),
+    FOREIGN KEY(patients_id) REFERENCES patients(id)
+ );
+ 
+CREATE TABLE treatments(
+    id serial PRIMARY KEY, 
+    type VARCHAR(50),
+    name VARCHAR(50)
+    );
```

Comment:
- [ ] As requested by the previous reviewer @zieeco  and according to the project requirements, you are required to create a join table for your project. He was kind enough to point out to your team exactly which tables need the join table. Kindly create the join table as was previously requested.

![image](https://user-images.githubusercontent.com/86318284/200776893-3f4aa67d-04af-4f3a-ab6a-ea3ed640014b.png)